zerver/migrations/0077_add_file_name_field_to_realm_emoji.py

```diff
@@ -125,7 +125,7 @@ def upload_to_s3(self, path, file_data, headers):
         key.set_contents_from_string(force_str(file_data), headers=headers)
 
     def upload_files(self, response, resized_image, dst_path_id):
-        # type: (Response, binary_type, Text) -> None
+        # type: (Response, Optional[binary_type], Text) -> None
```

Comment:
I'm not convinced this is the right fix; should `resize_emoji` really return `None`?